dotnet-deb/dotnet-deb.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0">
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFrameworks>netcoreapp2.1;netcoreapp3.0;net5.0</TargetFrameworks>
+    <TargetFrameworks>net5.0;net6.0</TargetFrameworks>
```

Comment:
Let's keep .NET Core 2.1 and 3.1 until they are end of life.

Reply:
The problem is, if we want to save .NET Core 2.1 and use .NET 6 - we have to use to many preprocessor directives, like
`#if NET5_0_OR_GREATER`

Reply:
Something like this:
![image](https://user-images.githubusercontent.com/50652041/128195377-cddac726-3d85-43ba-bea5-193279e553c7.png)

As I understood by description - Microsoft.Build 16.10 can work only with .NET Framework 4.7.2 or .NET 5 or higher
![image](https://user-images.githubusercontent.com/50652041/128200349-e8618ff8-917d-451c-b4e1-ff5a5d258693.png)

Reply:
Thanks for clarifying! In that case, I'd _really_ appreciate it if we can defer updating those dependencies a bit, and have this PR focus on updating the target frameworks.

I'm fine with updating dependencies which don't influence the list of supported .NET Core versions (e.g. the test SDKs or NerdBank.GitVersioning).

I'd be happy to drop .NET Core 2 support end of August, but dropping .NET Core 3 may need a bit more of convincing ;-).

Reply:
Ok, I will check all the references, and try to configure it for the .NET Core 3.1 and higher versions :-)